specification/peering/resource-manager/readme.md

```diff
@@ -81,6 +81,11 @@ See configuration in [readme.csharp.md](./readme.csharp.md)
 ## Go
 
 See configuration in [readme.go.md](./readme.go.md)
+
+## CLI
```

Comment:
It is interesting to me the CLI appeared here yet the power shell  not appeared. Will powershell also using this mark up?